src/main/java/com/ecore/roles/web/TeamsApi.java

```diff
@@ -0,0 +1,15 @@
+package com.ecore.roles.web;
+
+import com.ecore.roles.web.dto.TeamDto;
+import org.springframework.http.ResponseEntity;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface TeamsApi {
+
+    ResponseEntity<List<TeamDto>> getTeams();
```

Comment:
It could be just called "get" or "getAll", because it's inside of the teams api.